src/org/usfirst/frc/team467/robot/Autonomous/Actions.java

```diff
@@ -245,588 +264,132 @@ public static ActionGroup centerBasicSwitchRight() {
 		mode.addActions(start());
 		mode.addActions(move(4.0));
 		mode.addActions(turn(90));
-		mode.addActions(move(4.27)); 
+		mode.addActions(move(4.27));
 		mode.addActions(turn(-90));
 		mode.addActions(move(4.34));
 		mode.addAction(releaseCube());
+		mode.addAction(pauseGrabber());
 		return mode;
 	}
 
-	public static ActionGroup rightBasicSwitchRight() {
-		String actionGroupText = "Start on right, put cube on switch.";
-		ActionGroup mode = new ActionGroup(actionGroupText);
-		mode.addActions(start());
-		mode.addActions(move(12.33)); 
-		mode.addActions(turn(-90));
-		mode.addActions(move(1.479)); // 1' 5.75"
-		mode.addAction(releaseCube());
-		return mode;
-	}
-
-	//SCALE - BASIC
-
-	public static ActionGroup leftBasicScaleLeft() {
-		String actionGroupText = "Start on left, put cube on left scale";
-		ActionGroup mode = new ActionGroup(actionGroupText);
-		mode.addActions(start());
-		mode.addActions(move(25.33));
-		mode.addActions(turn(90));
-		mode.addActions(move(0.375));
-		mode.addAction(releaseCube());
-		return mode;
-	}
-
-	public static ActionGroup leftBasicScaleRight(){
-		String actionGroupText = "Start on left, put cube on right scale";
-		ActionGroup mode = new ActionGroup(actionGroupText);
-		mode.addActions(start());
-		mode.addActions(move(18.14));
-		mode.addActions(turn(90));
-		mode.addActions(move(19.08));
-		mode.addActions(turn(-90));
-		mode.addActions(move(7.19));
-		mode.addActions(turn(-90));
-		mode.addActions(move(0.375));
-		mode.addAction(releaseCube());
-		return mode;
-	}
-
-	public static ActionGroup centerBasicScaleLeft(){
-		String actionGroupText = "Start in center, put cube on left scale";
+	public static ActionGroup basicSwitchOurSide() {
+		String actionGroupText = "Put cube on our side switch.";
 		ActionGroup mode = new ActionGroup(actionGroupText);
 		mode.addActions(start());
-		mode.addActions(move(4.0));
+		mode.addActions(move(12.33));
 		mode.addActions(turn(-90));
-		mode.addActions(move(10.0));
-		mode.addActions(turn(90));
-		mode.addActions(move(21.33));
-		mode.addActions(turn(90));
-		mode.addActions(move(0.375));
+		mode.addActions(move(2.0)); 
 		mode.addAction(releaseCube());
+		mode.addAction(pauseGrabber());
 		return mode;
 	}
 
-	public static ActionGroup centerBasicScaleRight(){
-		String actionGroupText = "Start in center, put cube on right scale";
+	public static ActionGroup basicScaleOurSide(){
+		String actionGroupText = "Put cube on our side scale";
 		ActionGroup mode = new ActionGroup(actionGroupText);
 		mode.addActions(start());
-		mode.addActions(move(4.0));
-		mode.addActions(turn(90));
-		mode.addActions(move(9.08));
-		mode.addActions(turn(-90));
-		mode.addActions(move(21.33));
-		mode.addActions(turn(-90));
+		mode.addActions(move(23.33));
+		mode.addAction(elevatorToHighScale());
+		mode.addActions(move(2.0));
+		mode.addActions(turn(-95));
 		mode.addActions(move(0.375));
 		mode.addAction(releaseCube());
+		mode.addAction(pauseGrabber());
```

Comment:
All of these modes need to be carefully reviewed. I see a -95deg above, I don't think we want to keep that. Somebody who knows all these modes well needs to review each mode carefully.